src/main/java/seedu/address/model/Model.java

```diff
@@ -204,9 +204,15 @@
     void updateFilteredAppointmentList(Predicate<Appointment> predicate);
 
     /**
-     * Sorts the the filtered appointment list by the given {@code comparator}.
+     * Sorts the the appointment list by the given {@code comparator}.
```

Comment:
shouldn't it sort the filtered list?

Reply:
The method actually sorts the appointment list but the displayed filtered appointment list is sorted accordingly. Should I change back the javadoc comment?

Reply:
yeah, sure